Dev Extension for fragments Transitions

AlexDlaTorre opened the pull request:
Fragment+ext.kt file modified with function TransitionFragment

AlexDlaTorre committed:
Delete Gallery and SlideshowFragment

AlexDlaTorre committed:
Merge branch 'dev' into AleLayout

AlexDlaTorre committed:
Extension in class fragments

Extension for the transition from anActivity to a Fragment in file  Fragment+ext.kt

jpgpineda commented on app/src/main/java/com/example/bankodemia/ui/home/HomeFragment.kt:
```diff
@@ -28,10 +28,10 @@ class HomeFragment : Fragment() {
-        homeViewModel.text.observe(viewLifecycleOwner) {
-            textView.text = it
-        }
+        //val textView: TextView = binding.textHome
```
Si este codigo lo vas a utilizar mas adelante podrias añadir un comentario que diga: 
// TODO: (descripcion de que hara ese codigo despues)

AlexDlaTorre replied:
Este es el automático del navBar, todo ese se va quitar

jpgpineda commented on app/src/main/java/com/example/bankodemia/Core/Fragment+Ext.kt:
```diff
@@ -27,4 +32,17 @@ fun Fragment.showSnackBarActionMessage(message: String, actionTitle: String, dur
     }
+}
+
+fun TransitionFragment(button: Button,fragmentName: Fragment,supportFragmentManager: FragmentManager) {
```
Aqui para que la exntesion funcione debes poner:
 fun Fragment.transitionFragment(button: Button,fragmentName: Fragment,supportFragmentManager: FragmentManager) { 
 
 De otra forma seria solo una funcion externa
 
 Igual, recuerda que el nombre de las funciones debe empezar con minuscula

JoseSom commented on app/src/main/java/com/example/bankodemia/Core/Fragment+Ext.kt:
```diff
@@ -27,4 +32,17 @@ fun Fragment.showSnackBarActionMessage(message: String, actionTitle: String, dur
+}
+
+fun TransitionFragment(button: Button,fragmentName: Fragment,supportFragmentManager: FragmentManager) {
+    val buttonName = button
```
No veo que la variable buttonName se ocupe mas adelante, igual la de fragment = fragmentName. Creo que se puede reducir y quitar algunas lineas si se ponen directamente

AlexDlaTorre committed:
Update Fragment+Ext.kt

funciones en minúscula

AlexDlaTorre committed:
PR#10 Modification

created Activity+Ext, to include the extensions for activities, and add here the transitionFragment function

AlexDlaTorre committed:
Merge branch 'AleLayout' of https://github.com/AlexDlaTorre/bankodemia into AleLayout
